frontend/src/hooks/sails/showroom/global.d.ts

```diff
@@ -0,0 +1,80 @@
+import { ActorId, CodeId } from 'sails-js';
+
+declare global {
```

Comment:
Same as previous comment about `global.d.ts` - better to include those types into associated `lib.ts`.